### PR TITLE
add brazil to readme

### LINKED ISSUE #1342: Add Brazil to readme.txt

### PR DESCRIPTION
**Pull request by leonardola**
Fixes #1342

## Changes proposed in this Pull Request:
This PR adds brazil to the accepted countries. 
Also it updates the readme to Markdown. (But I am ok to revert it to .txt if it is necessary)

### CODE REVIEW

File: README.md
Comment (malithsen) [inline README.md]: We should probably use a smaller heading here
Screen Shot 2021-11-01 at 5 37 33 PM

**Review by malithsen — COMMENTED** [review]
This looks good to me @leonardola 
The version number field doesn't seem to be updated properly when it was converted to md. Let me know if this is intentional

**Comment by leonardola** [comment]
@malithsen can you please review again to accept the changes. I can't merge without at least one approval :/

**Review by malithsen — APPROVED** [review]
Looks good now, thanks for the update

**Pull request merged**
